core/src/main/java/org/apache/iceberg/RemoveSnapshots.java

```diff
@@ -320,9 +320,8 @@ public void commit() {
     LOG.info("Committed snapshot changes");
 
     if (cleanExpiredFiles) {
+      LOG.info("Cleaning up of expired files is enabled");
```

Comment:
I think logs are better when using active voice. Here, it would be "Cleaning up expired files (local, incremental)"

Reply:
updated as per your suggestion. Thanks.

Reply:
I think incremental is not always true, if it is disabled?  What do you think, to add the mode as a variable:
```
String cleanupMode = incrementalCleanup ? "incremental" : "reachable";
LOG.info("Cleaning up of expired files is enabled (local, %s)", cleanupMode);

```

Reply:
True. I have updated it now. Thanks for the review.